demo/Makefile

```diff
@@ -55,7 +55,9 @@ build:	copy-plugins build-registry
 #	
 # If using boot2docker, you need to tell your remote debugger to use the boot2docker VM ip (ala boot2docker ip).
 
-DOCKER_RUN=docker run --rm -p 127.0.0.1:8080:8080 -v $(shell which docker):/usr/bin/docker -v /var/run/docker.sock:/var/run/docker.sock --group-add=$(shell stat -c %g /var/run/docker.sock)
+STAT=$(firstword $(shell which gstat) $(shell which stat))
```

Comment:
On OSX, if you want to use GNU features, you need to install the homebrew versions of common tools like stat (which is then called gstat).  This should work on brew configured machines, as well as linux machines (which don't have gstat, as the normal stat is GNU-y).